### DISCUSSION TITLE
Nginx configuration

### DISCUSSION THREAD

**Discussion by ibravoh** [vercel/next.js#48467] [Help]
### Summary

Hi guys, I would appreciate if anyone can help me with a default nginx configuration for nexjs. I set output to "standalone" in my next.config file.

this is what i have currently in my nginx config
`server {
  listen 443;
  ssl_protocols TLSv1.2 TLSv1.3;
  server_tokens off;
  location / {
    root   /usr/share/nginx/html;
    index  index.html index.htm;
    try_files $uri $uri/ /index.html;
    expires 2d;
  }
  error_page   500 502 503 504  /50x.html;
  location = /50x.html {
    root   /usr/share/nginx/html;
  }
}`

this is my dockerfile configuration
` 
FROM node:14.20-alpine as build
WORKDIR /app
ENV PATH /app/node_modules/.bin:$PATH
COPY {name of my folder}/package.json /app/package.json
RUN apk update && \
    apk upgrade && \
    apk add git
RUN npm install --only=prod
RUN yarn
RUN yarn add react-scripts@3.4.0 -g
COPY {name of my folder}/ /app
RUN ls -la /app && cat /app/.env
RUN yarn build
# production environment
FROM nginx:1.16.0-alpine
COPY --from=build /app/build /usr/share/nginx/html
RUN rm /etc/nginx/conf.d/default.conf
COPY devops/projects/services/frontend/{name of my folder}/docker/nginx.conf /etc/nginx/conf.d
EXPOSE 443
CMD ["nginx", "-g", "daemon off;"]`

nexjs version : 13.2.4

currently is displays nginx default page instead of my app.

Thanks in advance

**Comment by ciruz** [comment]
nginx with ProxyPass and pm2 is no option?

Like let next.js run via pm2 in the background and use proxy pass to redirect all nginx requests to next.js:

https://episyche.com/blog/user-guide-for-deploying-the-nextjs-app-in-production-using-pm2-and-nginx

**Comment by yenkhoon** [comment]
is this issue resolved?

server {
    listen       80;
    server_name  localhost;

    access_log  /var/log/nginx/host.access.log  main;
    error_log /var/log/nginx/error.log warn;

    location / {
        root   /usr/share/nginx/html;
        index  index.html index.htm;
        try_files $uri $uri/ /index.html;
    }
    error_page   500 502 503 504  /50x.html;
    location = /50x.html {
        root   /usr/share/nginx/html;
    }

}

this is my nginx config, i gotten forbidden, i have copied the .next content into /usr/share/nginx/html
